recipes/mummer/build.sh

```diff
@@ -47,12 +49,13 @@ for i in exact-tandems dnadiff mapview mummerplot nucmer promer run-mummer1 run-
   perl -i -pe 's/(envs\/\_\_.*)(\K\@)/\\@/' $i
 done
 
+# fix hashbang lines to use conda's perl
+for i in dnadiff mapview mummerplot nucmer promer; do
```

Comment:
I moved this out for clarity and because I think the linux build actually needs this too, but the `file` command is not available on the build container.